orchagent/qosorch.cpp

```diff
@@ -616,15 +617,15 @@ sai_object_id_t QosOrch::initSystemAclTable()
     sai_status_t status;
 
     /* Create system ACL table */
-    attr.id = SAI_ACL_TABLE_ATTR_BIND_POINT;
-    attr.value.s32 = SAI_ACL_BIND_POINT_SWITCH;
+    attr.id = SAI_ACL_TABLE_ATTR_ACL_BIND_POINT_TYPE_LIST;
+    attr.value.s32 = SAI_ACL_BIND_POINT_TYPE_SWITCH;
```

Comment:
switch bind point not supported in mellanox implementation, as we believe it's not well defined
please use alternative bind point, such as binding to all ports